src/components/auditlog/AuditLogsTable.js

```diff
@@ -8,46 +8,44 @@ import UserLink from 'components/users/Link';
 const AuditLogsTable = ({ auditLog, hideUserColumns = false }) => {
     const numColumns = hideUserColumns ? 4 : 6;
     return (
-        <table>
-            <thead>
-                <tr>
-                    <th>Action</th>
-                    <th>IP address</th>
-                    <th>User agent</th>
-                    <th>Date/Time</th>
+        <Table size='sm'>
+            <Thead>
+                <Tr>
+                    <Th>Action</Th>
+                    <Th>IP address</Th>
+                    <Th>User agent</Th>
+                    <Th>Date/Time</Th>
                     {!hideUserColumns &&
                         <>
-                            <th>User</th>
-                            <th>Role</th>
+                            <Th>User</Th>
+                            <Th>Role</Th>
                         </>
                     }
-                    <th>Object</th>
-                </tr>
-            </thead>
-            <tbody>
+                    <Th>Object</Th>
+                </Tr>
+            </Thead>
+            <Tbody>
                 {auditLog.length === 0 && <NoResultsTableRow numColumns={numColumns} />}
                 {auditLog.map((entry, index) => {
                     return (
-                        <tr key={index}>
-                            <td>
-                                <Badge>{entry.action}</Badge>
-                            </td>
-                            <td><Ipv4Link value={entry.client_ip} /></td>
-                            <td>{entry.user_agent ? <UserAgentLabel userAgent={entry.user_agent} /> : '-'}</td>
-                            <td>{entry.insert_ts}</td>
+                        <Tr key={index}>
+                            <Td><Tag>{entry.action}</Tag></Td>
```

Comment:
The problem with using `Tag` is that is the same component I am using for tags in commands, vulnerabilities and so on and we don't want to confuse the user making them look like they are the same thing. (eg tags in commands are clickable and take you to the search page, this action not)

Reply:
![image](https://user-images.githubusercontent.com/220190/130297175-2205b527-bb88-4fa0-8fd7-d02cdec6982c.png)